File: src/com/kiwi/jujy/four/java/Button.java
package com.kiwi.jujy.four.java;

import com.kiwi.jujy.two.open.View;

/**
 * Copyright (C), 2017-2018, XXX有限公司
 * FileName: Button
 * Author:   kiwilss
 * Date:     2018/7/20 13:39
 * Description: button
 * History:
 * <author>          <time>          <version>          <desc>
 * 作者姓名           修改时间           版本号              描述
 */

public class Button extends View{
    public void setOnClickListener(OnClickListener listener){
        listener.onClick(this);
    }


}

File: src/com/kiwi/kotlin2/demo_class/Button.kt
package com.kiwi.kotlin2.demo_class



/**
 * Copyright (C), 2017-2018, XXX有限公司
 * FileName: Button
 * Author:   kiwilss
 * Date:     2018/4/24 16:19
 * Description: button
 * History:
 * <author>          <time>          <version>          <desc>
 * 作者姓名           修改时间           版本号              描述
 */
class Button : Clickable {
    override fun click() = println("i was clicked")

//     override fun showOff() {
//        super.showOff()
//    }
}

fun main(args: Array<String>) {
    Button().click()
    Button().showOff()
}